inc/saiport.h

```diff
@@ -3368,6 +3368,12 @@ typedef enum _sai_port_stat_t
     /** Packets trimmed due to failed shared buffer admission [uint64_t] */
     SAI_PORT_STAT_TRIM_PACKETS,
 
+    /** Packets trimmed but dropped due to failed shared buffer admission on a new queue (for trimmed packets) */
+    SAI_PORT_STAT_TRIM_DROPPED_PACKETS,
```

Comment:
We are following a fixed nomenclature like TRIM QUEUES, TRIM PACKET and so on.
Keeping consistent with it, please change the attr to
SAI_PORT_STAT_DROPPED_TRIM_PACKETS